docs/BR.md

```diff
@@ -1082,6 +1086,10 @@ CAs MAY check CAA records at any other time.
 
 When processing CAA records, CAs MUST process the issue, issuewild, and iodef property tags as specified in RFC 8659, although they are not required to act on the contents of the iodef property tag. Additional property tags MAY be supported, but MUST NOT conflict with or supersede the mandatory property tags set out in this document. CAs MUST respect the critical flag and not issue a certificate if they encounter an unrecognized property tag with this flag set.
 
+*Effective September 15, 2025*, when processing CAA records, CAs MUST process the accounturi and validationmethods parameters as specified in RFC 8657. In addition:
```

Comment:
RFC 8657 section 5.6 requires DNSSEC to be validated while performing CAA checks; under the current text, CAs would need to start validating DNSSEC for all CAA DNS queries; is this intentional?

We have seen that in practice many domains that are able to validate under current rules would begin to fail with strict DNSSEC enforcement. GTS would love to see DNSSEC used more widely and configured properly more often, but more time is likely needed before strict DNSSEC checks cause minimal issues for legitimate requesters.

Reply:
Thank you, this is an excellent point. We discussed this today on the Validation SC call and decided that the cleanest solution is to merge this ballot with @clintwilson's proposal to require DNSSEC as drafted at https://github.com/cabforum/servercert/compare/main..Require-DNSSEC-for-CAA-Lookups

Reply:
Clint's DNSSEC ballot has been incorporated.